### DISCUSSION TITLE
Is it possible to take data from outer file use to filter in inline (body editor)?

### DISCUSSION THREAD

**Discussion by Gourabh72** [mockoon/mockoon#1198] [Help]
I want to filter json external file, I want to data of that file in body editor.

**Comment by 255kb** [comment]
If you want to use an external file you can reference it in the "Body file" type: 
![image](https://github.com/mockoon/mockoon/assets/7489814/90bfa728-41bc-409a-9fda-ffb2fbe418fc)

[External files in JSON format support templating too](https://mockoon.com/docs/latest/templating/overview/#body-and-file-content-templating).

**Reply by janeivind** [reply]
I would like to reference the imported file, in the Body Inline block. In the same way as you can use data/dataRaw for data sets, something like
```
    {{{ stringify (
        filter
          (dataFile)
          (object status=(urlParam 'status'))
      )
    }}}
```
Where dataFile is reserved word and holder for the result of defined imported file.

**Reply by 255kb** [reply]
This is not possible and is not planned either. 
What you can do in external files though, is to reference data buckets.
